README.md

```diff
@@ -1 +1,36 @@
-# mediterranea-marine-nextjs
\ No newline at end of file
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
Remove the default README, add the readme sections from here that are applicable for now

https://github.com/abelbordonado/readme-and-templates/blob/main/readme/frontend-readme.md